app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
**`[optional]`** You can make the post to be arranged by the newest to the top. Add `default_scope -> { order(created_at: :desc) }`

Reply:
Thank you for the feedback. We have implemented the required changes.